.github/workflows/sanity-test.yaml

```diff
@@ -355,11 +356,100 @@ jobs:
         with:
           service: sharepoint
           kind: incremental
-          backup-args: '--site "${{ vars.CORSO_M365_TEST_SITE_URL }}"'
+          backup-args: '--site "${{ vars.CORSO_M365_TEST_SITE_URL }}" --data libraries'
           restore-args: '--folder ${{ env.RESTORE_DEST_PFX }}${{ steps.new-data-creation-sharepoint.outputs.result }}'
           restore-container: '${{ env.RESTORE_DEST_PFX }}${{ steps.new-data-creation-sharepoint.outputs.result }}'
           log-dir: ${{ env.CORSO_LOG_DIR }}
           with-export: true
+          category: libraries
+
+##########################################################################################################################################
+
+# Sharepoint Lists
+
+      # generate new entries for test
+      - name: SharePoint Lists - Create new data
+        id: new-data-creation-sharepoint-lists
+        timeout-minutes: 30
+        working-directory: ./src/cmd/factory
+        run: |
+          suffix=$(date +"%Y-%m-%d_%H-%M-%S")
+
+          go run . sharepoint lists  \
+            --site ${{ vars.CORSO_M365_TEST_SITE_URL }} \
+            --user ${{ env.TEST_USER }} \
+            --secondaryuser  ${{ env.CORSO_SECONDARY_M365_TEST_USER_ID }} \
+            --tenant ${{ secrets.TENANT_ID }} \
+            --destination ${{ env.RESTORE_DEST_PFX }}$suffix \
+            --count 4 |
+            awk 'NR > 1 {print $2}' | tr '\n' ',' | sed -e 's/,$//' -e 's/^/result=/' |
+            tee $GITHUB_OUTPUT
+
+      - name: SharePoint Lists - Store restore container
+        id: sharepoint-lists-store-restore-container
+        run: |
+          echo ${{ steps.new-data-creation-sharepoint-lists.outputs.result }} |
```

Comment:
Converts 

`Corso_Test_Sanity_dt_tm_543beed0-aa14-4780-b5b7-cc87fba12f38,Corso_Test_Sanity_dt_tm_697c0cc1-60da-440c-9e96-65c7ba430098,Corso_Test_Sanity_dt_tm_297d0b08-13c4-4f13-82cb-000fcb33f47b,Corso_Test_Sanity_dt_tm_f02f0397-c371-4584-97b2-afe8fa8d6e14`

To `Corso_Test_Sanity_dt_tm`

Reply:
same here, a comment would be helpful.